xarray/tests/test_dask.py

```diff
@@ -150,7 +150,9 @@ def test_repr(self):
         expected = dedent(
             """\
             <xarray.Variable (x: 4, y: 6)>
-            dask.array<array, shape=(4, 6), dtype=float64, chunksize=(2, 2)>"""
+            {!r}""".format(
+                self.lazy_var.data
+            )
```

Comment:
If these get any more complex, I've had great results using https://gitlab.com/uweschmitt/pytest-regtest - a snapshot / expect test framework - for string outputs that require checking but not explicitly constructing.